gulpfile.js

```diff
@@ -1,32 +1,87 @@
+// var gulp = require('gulp'),
```

Comment:
There should be a .editorconfig  file created with the recommended indentations so that it will be consistent across editors.